openvpn/config.sls

```diff
@@ -8,6 +8,14 @@ include:
 {% if type in ['client', 'server', 'peer'] %}
 {% for name, config in names.iteritems() %}
 
+{% macro _permissions(mode=644, user=None, group=None) %}
```

Comment:
I' curious: Why `_permissions` and not `permissions`?

Reply:
According to the Jinja documentation ```If a macro name starts with an underscore, it’s not exported and can’t be imported.```
I think this macro wouldn't be used elsewhere, but maybe this is just being fussy.